koji/vars/users.yml

```diff
@@ -109,17 +74,38 @@ enabled_users:
       - admin
       - build
       - repo
-  - name: semyers
-    permissions:
-      - build
-      - repo
-  - name: stbenjam
-    permissions:
-      - admin
   - name: zhunting
     permissions:
       - admin
       - build
       - repo
 
 disabled_users: []
+  - name: adprice
+    permissions: []
```

Comment:
It feels odd that you have to specify permissions for a disabled user. Can't we default that to an empty array?

Reply:
Fair thought, I hard set that in the playbook here https://github.com/theforeman/foreman-infra/pull/1625